components/x-teaser/src/Title.jsx

```diff
@@ -15,7 +15,7 @@ export default ({ title, altTitle, headlineTesting, relativeUrl, url, indicators
 				{' '}
 			</Link>
 			{indicators && indicators.accessLevel === 'premium' ? (
-				<span className="o-labels o-labels--premium" aria-label="Premium content">
+				<span className="o-labels o-labels--content-premium" aria-label="Premium content">
```

Comment:
Do you think we should keep the old class name as well to smooth over the transition for apps which haven't had o-labels bumped, or not do that and bump apps as and when we see display issues?

Reply:
Good idea, I think having both in won't cause issues apart from slightly messy HTML. Otherwise we could make this a breaking change, which might be a bit extreme.

Reply:
We haven't released v1 yet so breaking slightly is OK IMO.

Reply:
Ok, shall I remove the messy HTML then?

Reply:
I think this is good and it's commented so 👍